pandas/tests/apply/test_frame_apply.py

```diff
@@ -61,6 +63,7 @@ def test_apply(float_frame, engine, request):
         assert result.index is float_frame.index
 
 
+@pytest.mark.xfail(using_string_dtype(), reason="TODO(infer_string)", strict=False)
```

Comment:
All all the `strict=False` strictly necessary in this PR?

Reply:
Yeah, I only added it when necessary, and it's only added to parametrized tests where some of the cases were xpassing and some xfailing.

In theory you could mark individual cases as xfail. But since all those xfails are meant to be resolved and fixed in the comings weeks, I don't want to do that additional somewhat useless effort ..